pyproject.toml

```diff
@@ -49,6 +45,10 @@ pre-commit = [
 testing = [
     "pytest ~=7.2",
     "pytest-cov ~=4.0",
+    "EMMOntoPy ~=0.4",
+    "rdflib ~=6.2",
+    "SPARQLWrapper ~=2.0",
+    "DLite-Python >=0.3.15",
```

Comment:
These should be replicated in the `dev` extra as well (like the `docs` extras are replicated both under `docs` and `dev`.

Either that, or the `testing` extra should be added to the `ci_tests.yml` file's `install_extras` input, i.e., its value should be `[dev,testing]` instead of just `[dev]`.

Reply:
Thanks, that did it :-)

Reply:
It would be great to have things documented somewhere...

Reply:
The callable workflows are documented at https://SINTEF.github.io/ci-cd.
However, this particular duplication "issue" is a quirk of `pyproject.toml` being a static file and the desire to have an installable extra that is a "catch-em-all".